### PR TITLE
BHBC-1290_DM

### PR DESCRIPTION
**Pull request by cgarrettjones**
# Overview

## Links to jira tickets

- [BHBC-1290](https://quartech.atlassian.net/browse/BHBC-1290)

## This PR contains the following changes

- data model changes associated with BHBC-1290
- added table submission_message_class
- adjusted first pass polutation scripts for submission_message_class
- adjusted first pass polutation scripts for submission_message_type

## This PR contains the following types of changes

- [ ] New feature (change which adds functionality)
- [ ] Enhancement (improvements to existing functionality)
- [ ] Bug fix (change which fixes an issue)
- [ ] Misc cleanup / Refactoring / Documentation
- [ ] Version change

## Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### General

- [ ] I have performed a self-review of my own code

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Screenshots

Please add any relevant UI screenshots, if applicable.

### CODE REVIEW

**Review by sdevalapurkar — DISMISSED** [review]
code lgtm

File: database/src/migrations/release.0.23/populate_submission_message_type.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- populate_submission_message_type.sql
+
+-- term related
+insert into submission_message_type (name, record_effective_date, description, submission_message_class_id) values ('Duplicate Header', now(), 'A duplicate header term was detected.', (select submission_message_class_id from submission_message_class where name = 'Error'));
+insert into submission_message_type (name, record_effective_date, description, submission_message_class_id) values ('Unknown Header', now(), 'An unknown header term was detected.', (select submission_message_class_id from submission_message_class where name = 'Warning'));
+insert into submission_message_type (name, record_effective_date, description, submission_message_class_id) values ('Missing Required Header', now(), 'A required header term was not detected.', (select submission_message_class_id from submission_message_class where name = 'Error'));
+insert into submission_message_type (name, record_effective_date, description, submission_message_class_id) values ('Missing Recommended Header', now(), 'A recommended header term was ` detected.', (select submission_message_class_id from submission_message_class where name = 'Warning'));
```
Comment (NickPhura) [inline database/src/migrations/release.0.23/populate_submission_message_type.sql]:
```suggestion
insert into submission_message_type (name, record_effective_date, description, submission_message_class_id) values ('Missing Recommended Header', now(), 'A recommended header term was detected.', (select submission_message_class_id from submission_message_class where name = 'Warning'));

```

Comment (NickPhura) [reply database/src/migrations/release.0.23/populate_submission_message_type.sql]: There is a rogue backtick in the message.  Probably just a miss-type.

**Comment by NickPhura** [comment]
Looks good!  We just need to hold off on merging temporarily so we can make sure these changes and Anissa's changes don't collide.

**Review by anissa-agahchen — APPROVED** [review]
Made appropriate API and front-end changes for this to all work.

**Pull request merged**